lib/themes/dosomething/paraneue_dosomething/js/reportback/Reportback.js

```diff
@@ -1,301 +1,284 @@
-    enableResponsive: function () {
-      var _this = this;
+  /**
+   * Enable Media Query check to shift around markup.
+   */
+  enableResponsive: function() {
+    const _this = this;
 
-      $(window).on('resize', debounce(function () {
-        if (_this.isLargerMedia()) {
-          if (_this.$viewMoreContainer) {
-            _this.$viewMoreContainer.detach();
-            _this.$viewMoreContainer.insertAfter(_this.$reportbackWrapper);
-          }
-        } else {
-          if (_this.$viewMoreContainer) {
-            _this.$viewMoreContainer.detach();
-            _this.$viewMoreContainer.insertAfter(_this.$initialGallery);
-          }
+    $(window).on('resize', debounce(function() {
+      if (_this.isLargerMedia()) {
+        if (_this.$viewMoreContainer) {
+          _this.$viewMoreContainer.detach();
+          _this.$viewMoreContainer.insertAfter(_this.$reportbackWrapper);
+        }
+      } else {
+        if (_this.$viewMoreContainer) {
+          _this.$viewMoreContainer.detach();
+          _this.$viewMoreContainer.insertAfter(_this.$initialGallery);
         }
-      }, 300));
-    },
+      }
+    }, 300));
+  },
 
 
-    /**
-     * Add the View More button if there are additional reportbacks.
-     */
-    enableViewMore: function () {
-      var _this = this;
-      var buttonText = Drupal.t('View more');
+  /**
+   * Add the View More button if there are additional reportbacks.
+   */
+  enableViewMore: function() {
+    const _this = this;
+    const buttonText = Drupal.t('View more');
 
-      this.$viewMoreContainer = $(`<div class="reportback__view-more"></div>`);
-      this.$viewMoreButton = $(`<button id="view-more-button" class="button button--view-more -tertiary inline-alt-text-color">${buttonText}</button>`);
+    this.$viewMoreContainer = $(`<div class="reportback__view-more"></div>`);
+    this.$viewMoreButton = $(`<button id="view-more-button" class="button button--view-more -tertiary inline-alt-text-color">${buttonText}</button>`);
 
-      this.$viewMoreContainer.append(this.$viewMoreButton);
+    this.$viewMoreContainer.append(this.$viewMoreButton);
 
-      if (this.isLargerMedia()) {
-        this.$viewMoreContainer.insertAfter(this.$reportbackWrapper);
-      }
-      else {
-        this.$viewMoreContainer.insertAfter(this.$initialGallery);
-      }
+    if (this.isLargerMedia()) {
+      this.$viewMoreContainer.insertAfter(this.$reportbackWrapper);
+    } else {
+      this.$viewMoreContainer.insertAfter(this.$initialGallery);
+    }
 
-      this.$viewMoreButton.on('click', function () {
-        _this.request();
-      });
-    },
+    this.$viewMoreButton.on('click', function() {
+      _this.request();
+    });
+  },
 
 
-    /**
-     * Initialize the ImageUpload interface.
-     */
-    imageUploadInit: function () {
-      new ImageUpload(this.$uploadButton);
+  /**
+   * Initialize the ImageUpload interface.
+   */
+  imageUploadInit: function() {
+    imageUpload(this.$uploadButton);
```

Comment:
Hmm, is this from [this rule](http://eslint.org/docs/rules/no-new)? I think if we remove the `new` operator, all the references to `this` inside the `ImageUpload` function will refer to the window object, rather than being stored on the image uploader itself. I think a good solution for now would be to store `new ImageUpload` to a property on the `Reportback` object.